### PR TITLE
Add script to setup docker-daemon for development container.

### PR DESCRIPTION
**Pull request by ydye**

### CODE REVIEW

File: kubernetes-deployment/container-setup.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+cp -r /docker/* /usr/bin/
+docker &
```
Comment (sterowang) [inline kubernetes-deployment/container-setup.sh]: This line looks unnecessary from my experience.

File: kubernetes-deployment/container-setup.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+cp -r /docker/* /usr/bin/
```
Comment (sterowang) [inline kubernetes-deployment/container-setup.sh]: Do you think is it properly to add this as a COPY statement in Docker file?

File: kubernetes-deployment/container-setup.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+docker &
+docker run hello-world
+
+pip install kubernetes
```
Comment (sterowang) [inline kubernetes-deployment/container-setup.sh]: Also move this into Dockerfile?

File: kubernetes-deployment/container-setup.sh [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+cp -r /docker/* /usr/bin/
+docker &
+docker run hello-world
```
Comment (sterowang) [inline kubernetes-deployment/container-setup.sh]: If run hello-world failed, what will we do then?

File: kubernetes-deployment/container-setup.sh (hunk 1 above)
Comment (ydye) [reply kubernetes-deployment/container-setup.sh]: Removed

**Pull request merged**